src/schemaTypeResolver.ts

```diff
@@ -91,7 +91,9 @@ export const schemaFromConfluentSchema = (
         break
       }
       default:
-        throw new ConfluentSchemaRegistryArgumentError('invalid schemaType')
+        throw new ConfluentSchemaRegistryArgumentError(
+          `invalid schemaType ${(confluentSchema as ConfluentSchema).type}`,
```

Comment:
Why would you need to cast this when `confluentSchema` is already a `ConfluentSchema` according to the function parameter definition?